### PR TITLE
Multiorderer merge

### PR DESCRIPTION
**Pull request by jagpreetsinghsasan**
#154

### CODE REVIEW

File: platforms/hyperledger-fabric/charts/peernode/templates/configmap.yaml [hunk 1]
```diff
@@ -29,7 +29,8 @@ data:
   CORE_PEER_GOSSIP_USELEADERELECTION: "false"
   CORE_PEER_GOSSIP_ORGLEADER: "true"
   CORE_PEER_PROFILE_ENABLED: "true"
-  CORE_PEER_ADDRESSAUTODETECT: "false"
+  CORE_PEER_ADDRESSAUTODETECT: "true"
+
```
Comment (sownak) [inline platforms/hyperledger-fabric/charts/peernode/templates/configmap.yaml]: remove space. 
Also change 
CORE_PEER_GOSSIP_USELEADERELECTION: "true"
CORE_PEER_GOSSIP_ORGLEADER: "false"

File: platforms/hyperledger-fabric/configuration/deploy-network.yaml [hunk 2]
```diff
@@ -192,11 +193,11 @@
         participants: "{{ item.participants }}"
       loop: "{{ network['channels'] }}"
 
-    # delete build directory
-    - name: Remove build directory
-      file:
-        path: "./build"
-        state: absent
+    # # delete build directory
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/deploy-network.yaml]: remove comment

File: platforms/hyperledger-fabric/configuration/roles/create/channels/tasks/orderercheck.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    context: "{{ orderer_context }}"
+    label_selectors:
+      - app = {{ orderer.name }}
+  loop: "{{ ordererorg.services.orderers }}"
+  loop_control:
+    loop_var: orderer
+  register: get_orderer
+  until: ("Running" in get_orderer|json_query('resources[*].status.phase'))
+  retries: "{{ network.env.retry_count}}"
+  delay: 40
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/roles/create/channels/tasks/orderercheck.yaml]: add newline at end of file

File: platforms/hyperledger-fabric/configuration/samples/network-fabricv2.yaml [hunk 4]
```diff
@@ -145,12 +151,19 @@ network:
           grpc:
             port: 9092
                 
-        orderer:
-          name: orderer
-          type: orderer          
+        orderers:
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/samples/network-fabricv2.yaml]: Add a comment that "# This sample has multiple orderers as an example. You can use a single orderer for most production implementations."

File: platforms/hyperledger-fabric/scripts/generate-crypto.sh [hunk 5]
```diff
@@ -110,4 +110,43 @@ while [ ${COUNTER} -lt ${NO_OF_PEERS} ]; do
 	let COUNTER=COUNTER+1
 done
 
+while [ ${COUNTER} -lt ${NO_OF_PEERS} ] && [ "$1" != "peer" ]; do
```
Comment (sownak) [inline platforms/hyperledger-fabric/scripts/generate-crypto.sh]: is this change done for the generate-crypto tpl file?

File: platforms/shared/configuration/site.yaml [hunk 6]
```diff
@@ -8,8 +8,8 @@
   - import_playbook: kubernetes-env-setup.yaml         # Then deploy Flux for each organization
   - import_playbook: "{{ playbook_dir }}/../../hyperledger-fabric/configuration/deploy-network.yaml"
     when: network.type == 'fabric' 
-  - import_playbook: "{{ playbook_dir }}/../../hyperledger-fabric/configuration/chaincode-install-instantiate.yaml"
-    when: network.type == 'fabric' 
+  # - import_playbook: "{{ playbook_dir }}/../../hyperledger-fabric/configuration/chaincode-install-instantiate.yaml"
```
Comment (sownak) [inline platforms/shared/configuration/site.yaml]: remove comments

File: platforms/hyperledger-fabric/scripts/generate-crypto.sh (hunk 5 above)
Comment (jagpreetsinghsasan) [reply platforms/hyperledger-fabric/scripts/generate-crypto.sh]: No. This change is just for the script as the same script is used both by the orderer and the peer and shall not include checks like $1 = 'orderer' as 'orderer' here is the name of the orderer and not the type.

File: platforms/hyperledger-fabric/configuration/samples/network-fabricv2.yaml (hunk 4 above)
Comment (sownak) [reply platforms/hyperledger-fabric/configuration/samples/network-fabricv2.yaml]: This is not addressed

**Pull request merged**